frame/staking/src/pallet/impls.rs

```diff
@@ -553,10 +552,10 @@ impl<T: Config> Pallet<T> {
 						total = total.saturating_add(stake);
 					});
 
-				let exposure = Exposure { own, others, total };
+				let exposure = Exposure { own, others: others.try_into().expect("TODO"), total };
```

Comment:
This is exactly https://github.com/paritytech/substrate/pull/11585/files#r925580489